library/trakt.go

```diff
@@ -648,7 +648,7 @@ func updateShowWatched(xbmcHost *xbmc.XBMCHost, s *trakt.WatchedShow, watched bo
 
 // RefreshTraktCollected ...
 func RefreshTraktCollected(xbmcHost *xbmc.XBMCHost, itemType int, isRefreshNeeded bool) error {
-	if config.Get().TraktToken == "" || !config.Get().TraktSyncCollections {
+	if config.Get().TraktToken == "" {
```

Comment:
What is the idea of removing the use of a setting that was added for this case? 
That was the only place for those properties `TraktSyncCollections`, `TraktSyncWatchlist`, `TraktSyncUserlists`?
Idea was to disable sync for specific parts instead of disabling whole sync.

Reply:
Those settings are not removed, but moved inside functions that actually does trakt update. So we can do update of trakt and then if needed - also do kodi library update.